### PR TITLE
Add HelloWeb sample

### PR DESCRIPTION
**Pull request by abhinayagarwal**
Adds sample for web target

### CODE REVIEW

File: HelloWeb/README.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+## Basic Requirements
+
+A list of the basic requirements can be found online in the [Gluon documentation](https://docs.gluonhq.com/#_requirements).
+
+## Quick instructions
+
+### Run the sample
+
+    mvn javafx:run
```
Comment (jperedadnr) [inline HelloWeb/README.md:13]: Probably better `gluonfx:run` (for consistency with other samples)

File: HelloWeb/pom.xml [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+        <repository>
+            <id>gluon-nexus</id>
+            <url>https://nexus.gluonhq.com/nexus/content/repositories/releases/</url>
+        </repository>
+    </repositories>
+
+    <pluginRepositories>
+        <pluginRepository>
+            <id>gluon-nexus</id>
+            <url>https://nexus.gluonhq.com/nexus/content/repositories/releases/</url>
```
Comment (jperedadnr) [inline HelloWeb/pom.xml:70]: not needed anymore?

File: HelloWeb/src/main/java/com/gluonhq/sample/HelloWeb.java [hunk 3]
```diff
@@ -0,0 +1,475 @@
...
+import javafx.scene.shape.Line;
+import javafx.scene.shape.Rectangle;
+import javafx.scene.transform.Rotate;
+import javafx.scene.transform.Scale;
+import javafx.stage.Stage;
+import javafx.util.Duration;
+
+public class HelloWeb extends Application {
+
+    private final static int UTC = 2; // fix
```
Comment (jperedadnr) [inline HelloWeb/src/main/java/com/gluonhq/sample/HelloWeb.java:39]: This is probably not easy to fix, but can you check if there is a way to retrieve the timezone that works on web?

File: HelloWeb/src/main/resources/META-INF/native-image/filter-file.json [hunk 4]
```diff
@@ -0,0 +1,8 @@
+{ "rules": [
```
Comment (jperedadnr) [inline HelloWeb/src/main/resources/META-INF/native-image/filter-file.json:1]: I don't think we need any of the native-image config files from runagent, do we?

File: HelloWeb/src/main/resources/com/gluonhq/sample/styles.css [hunk 5]
```diff
@@ -0,0 +1,4 @@
+.root {
```
Comment (jperedadnr) [inline HelloWeb/src/main/resources/com/gluonhq/sample/styles.css:1]: Css is commented out, probably better to remove the file?

File: HelloWeb/README.md [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+
+### Run the sample in browser
+
+    mvn gluonfx:build gluonfx:nativerun -Pweb
```
Comment (jperedadnr) [inline HelloWeb/README.md:17]: Do we need to mention that Chrome should be default browser? Else maybe:
```
mvn gluonfx:build -Pweb

```
and then open `target/gluonfx/x86_64-web/gvm/web/index.html` with Chrome?

File: HelloWeb/src/main/resources/META-INF/native-image/filter-file.json (hunk 4 above)
Comment (abhinayagarwal) [reply HelloWeb/src/main/resources/META-INF/native-image/filter-file.json:1]: This application can also run on Desktop and these files are required?

File: HelloWeb/README.md [hunk 7]
```diff
@@ -0,0 +1,29 @@
...
+
+    mvn gluonfx:run
+
+### Run the sample in browser
+
+    mvn gluonfx:build gluonfx:nativerun -Pweb
+
+NOTE: Gluon currently supports Chrome, Chromium, and Firefox browser.
+
+### Desktop
```
Comment (johanvos) [inline HelloWeb/README.md:21]: Is this relevant here?

File: HelloWeb/README.md [hunk 8]
```diff
@@ -0,0 +1,29 @@
+# HelloWeb Application
+
+Gluon Application on the browser
+
+## Basic Requirements
+
+A list of the basic requirements can be found online in the [Gluon documentation](https://docs.gluonhq.com/#_requirements).
+
```
Comment (johanvos) [inline HelloWeb/README.md:8]: Although we don't use GraalVM for this atm, the compile step fails if GraalVM is not installed. Either we should remove the restriction in Gluon Substrate, or document here that GraalVM is needed (for those people who don't follow links)?

File: HelloWeb/README.md (hunk 7 above)
Comment (abhinayagarwal) [reply HelloWeb/README.md:21]: All our other samples work natively on Desktop, so does this sample.

File: HelloWeb/README.md (hunk 8 above)
Comment (abhinayagarwal) [reply HelloWeb/README.md:8]: `nativerun` which also has no dependency fails if `GRAALVM_HOME` is not defined. We need to file issues and take care of it in the plugin instead.

File: HelloWeb/README.md (hunk 7 above)
Comment (johanvos) [reply HelloWeb/README.md:21]: Yes, but I doubt this is relevant.
When reading this, it seems like these are the next steps to deploy on web. It is not clear that this has nothing to do with running on the web.
In this light, this sample is different from the other samples we have.

Comment (abhinayagarwal) [reply HelloWeb/README.md:21]: In that case, I will remove it and also the related files from `META-INF`

File: HelloWeb/src/main/resources/META-INF/native-image/filter-file.json (hunk 4 above)
Comment (abhinayagarwal) [reply HelloWeb/src/main/resources/META-INF/native-image/filter-file.json]: I have removed instructions for Desktop and related files.

**Review by johanvos — APPROVED** [review]

**Pull request merged**